picard/ui/infodialog.py

```diff
@@ -264,15 +264,15 @@ def format_file_info(file_):
     if '~sample_rate' in file_.orig_metadata:
         info.append((_('Sample rate:'), '%s Hz' % file_.orig_metadata['~sample_rate']))
     if '~bits_per_sample' in file_.orig_metadata:
-        info.append((_('Bits per sample:'), string_(file_.orig_metadata['~bits_per_sample'])))
+        info.append((_('Bits per sample:'), str(file_.orig_metadata['~bits_per_sample'])))
     if '~channels' in file_.orig_metadata:
         ch = file_.orig_metadata['~channels']
         if ch == 1:
             ch = _('Mono')
         elif ch == 2:
             ch = _('Stereo')
         else:
-            ch = string_(ch)
+            ch = str(ch)
```

Comment:
I looked at this recently, this code is broken right now. `ch` is always a `str` so the comparisson with integers fails. We don't need the `str()` here, but the comparisson needs to be changed to strings (`"1"`)

Reply:
But do we need such fancy conversion to "Mono" or "Stereo" at all ? Just displaying the number of channels is ok, that's just informative.

Reply:
Maybe not, even though I find it is a nice touch. I also looked into visualizing the channel mapping data we find in some FLACs, and that would give more verbose strings (like "5.1 (side)") we could display here. So I would for now keep it but fix it.

Reply:
Simplified and fixed by https://github.com/metabrainz/picard/pull/958/commits/0915fad10f66053c5a6bbaa4d2be0d773bce5c27

Reply:
It will be fixed in another PR.
https://tickets.metabrainz.org/projects/PICARD/issues/PICARD-1340